src/v/kafka/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ v_cc_library(
     Seastar::seastar
     v::bytes
     v::rpc
+    v::coproc
```

Comment:
This circular dependency between kafka and coproc should be broken. Maybe not in this PR, but soon.